app/kubemci/pkg/gcp/healthcheck/healthchecksyncer.go

```diff
@@ -143,12 +145,19 @@ func (h *HealthCheckSyncer) DeleteHealthChecks(ports []ingressbe.ServicePort) er
 func (h *HealthCheckSyncer) deleteHealthCheck(port ingressbe.ServicePort) error {
 	name := h.namer.HealthCheckName(port.Port)
 	glog.V(2).Infof("Deleting health check %s", name)
-	if err := h.hcp.DeleteHealthCheck(name); err != nil {
-		glog.V(2).Infof("Error in deleting health check %s: %s", name, err)
-		return err
+	err := h.hcp.DeleteHealthCheck(name)
+	if err != nil {
+		if utils.IsHTTPErrorCode(err, http.StatusNotFound) {
+			fmt.Println("Health check", name, "does not exist. Nothing to delete")
+			return nil
+		} else {
+			glog.V(2).Infof("Error in deleting health check %s: %s", name, err)
+			return err
+		}
+	} else {
+		glog.V(2).Infof("Successfully deleted health check %s", name)
```

Comment:
same as above. change to fmt.Println

Reply:
i'll skip- we'll say we deleted them all in the calling function.